### PR TITLE
Add link between stock inventory and lots

### LINKED ISSUE #4085: Link from "Articles in Stock" to "Lots in Stock"
We have a registry of "Articles in Stock" which looks like this:

image

It would be awesome if we could link directly from each of these inventory rows to the "Lots in Stock", filtering on the inventory item.  That way, a user can really easily figure out what lots are in stock for each inventory item.

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR is related to the issue #4085 

Close #4085

### CODE REVIEW

**Comment by kwilu** [comment]
Hello @mbayopanda, it seems like you have referenced #4085 in your pull request description, but you have not referenced them in your commit message description(s). When you reference an issue in a commit message, it automatically closes the corresponding issue when the commit is merged.

Please run `git commit --amend` in your command line client to amend your commit message description with `Fixes #4085.`.

An example of a correctly-formatted commit:
```
commit fabd5e450374c8dde65ec35f02140383940fe146
Author: kwilu
Date:   Sat Mar 18 13:42:40 2017 -0700

    pull requests: Check PR commits reference when issue is referenced.

    Fixes #51.
```

Thank you for your contributions to BHIMA!

**Comment by kwilu** [comment]
Hello @mbayopanda, it seems like you have referenced #4085 in your pull request description, but you have not referenced them in your commit message description(s). When you reference an issue in a commit message, it automatically closes the corresponding issue when the commit is merged.

Please run `git commit --amend` in your command line client to amend your commit message description with `Fixes #4085.`.

An example of a correctly-formatted commit:
```
commit fabd5e450374c8dde65ec35f02140383940fe146
Author: kwilu
Date:   Sat Mar 18 13:42:40 2017 -0700

    pull requests: Check PR commits reference when issue is referenced.

    Fixes #51.
```

Thank you for your contributions to BHIMA!

File: client/src/modules/stock/lots/registry.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ function StockLotsController(
   function startup() {
     if ($state.params.filters.length) {
       stockLotFilters.replaceFiltersFromState($state.params.filters);
-      stockLotFilters.cache.formatCache();
```
Comment (jniles) [inline client/src/modules/stock/lots/registry.js:95]: :dancer:

File: client/src/modules/stock/inventories/templates/inventory.cell.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<div class="ui-grid-cell-contents">
+  <a data-method="stock-lots"
+    ui-sref="stockLots({ filters : [
+      { key : 'inventory_uuid', value : row.entity.inventory_uuid, displayValue : row.entity.text, cacheable: false },
+      { key : 'depot_uuid', value : row.entity.depot_uuid, displayValue : row.entity.depot_text, cacheable: false },
+      { key : 'limit', value : 1000, cacheable: false }
```
Comment (jniles) [inline client/src/modules/stock/inventories/templates/inventory.cell.html:6]: I had totally forgotten about the `cacheable` flag.  Cool!

**Review by jniles — APPROVED** [review]
Tested and working.  Well done!

**Pull request merged**